src/client/datascience/jupyter/kernels/kernelCommandListener.ts

```diff
@@ -191,31 +182,13 @@ export class KernelCommandListener implements IDataScienceCommandListener {
                 undefined,
                 exc
             );
-            if (exc instanceof JupyterKernelPromiseFailedError && kernel) {
```

Comment:
This is no longer necessary, yaya, this error cannto happen (comment futher up in the PR)